source/infrastructure/hosting/azure-cip/index.html.md.erb

```diff
@@ -34,6 +34,34 @@ The new user will receive an invitation by email. Then the service managers can
 
 To access Azure Devops, the new user must access [Azure DevOps](https://dev.azure.com/dfe-ssp/). This will register them in the system then the service owner can add them to the project.
 
+## Privileged Identity Management (PIM) Requests
+
+Privileged Identity Management (PIM) is a service in Azure Active Directory (Azure AD) that enables you to manage, control, and monitor access to important resources at DfE such as access to staging and production environments.
```

Comment:
Maybe link to PIM: https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-configure

Reply:
Done